Product/Production/Services/DocumentRetrieveCore/src/test/java/gov/hhs/fha/nhinc/docretrieve/audit/transform/DocRetrieveAuditTransformsTest.java

```diff
@@ -0,0 +1,358 @@
+    protected AuditTransforms<RetrieveDocumentSetRequestType, RetrieveDocumentSetResponseType> getAuditTransforms() {
+        return new DocRetrieveAuditTransforms();
+    }
+
+    /**
+     * This method is override in this test class in order to check RD specific testing
+     *
+     * @param request
+     * @param isRequesting
+     * @param localIP
+     * @param webContextProperties
+     * @param remoteObjectIP
+     * @throws UnknownHostException
+     */
+    protected void testGetActiveParticipantSource(LogEventRequestType request, Boolean isRequesting, String localIP,
+        Properties webContextProperties, String remoteObjectIP) throws UnknownHostException {
+
+        ActiveParticipant sourceActiveParticipant = null;
+        List<ActiveParticipant> activeParticipant = request.getAuditMessage().getActiveParticipant();
+        for (ActiveParticipant item : activeParticipant) {
+            if (item.getRoleIDCode().get(0).getDisplayName() != null && item.getRoleIDCode().get(0).getDisplayName().
+                equals(AuditTransformsConstants.ACTIVE_PARTICIPANT_ROLE_CODE_SOURCE_DISPLAY_NAME)) {
+                sourceActiveParticipant = item;
+            }
+        }
+
+        if (isRequesting) {
+            assertEquals(remoteObjectIP, sourceActiveParticipant.getUserID());
+        }
+
+        if (isRequesting) {
+            assertEquals(ManagementFactory.getRuntimeMXBean().getName(), sourceActiveParticipant.getAlternativeUserID());
+        }
+        assertEquals(isRequesting, sourceActiveParticipant.isUserIsRequestor());
+        if (isRequesting) {
+            assertEquals(localIP, sourceActiveParticipant.getNetworkAccessPointID());
+        } else {
+            assertEquals(webContextProperties.getProperty(NhincConstants.REMOTE_HOST_ADDRESS),
+                sourceActiveParticipant.getNetworkAccessPointID());
+        }
+        assertEquals(AuditTransformsConstants.NETWORK_ACCESSOR_PT_TYPE_CODE_NAME,
+            sourceActiveParticipant.getNetworkAccessPointTypeCode());
+        assertEquals(AuditTransformsConstants.ACTIVE_PARTICIPANT_ROLE_CODE_SOURCE,
+            sourceActiveParticipant.getRoleIDCode().get(0).getCode());
+        assertEquals(AuditTransformsConstants.ACTIVE_PARTICIPANT_CODE_SYSTEM_NAME,
+            sourceActiveParticipant.getRoleIDCode().get(0).getCodeSystemName());
+        assertEquals(AuditTransformsConstants.ACTIVE_PARTICIPANT_ROLE_CODE_SOURCE_DISPLAY_NAME,
+            sourceActiveParticipant.getRoleIDCode().get(0).getDisplayName());
+    }
+
+    /**
+     * This method is override in this test class in order to check RD specific testing
+     *
+     * @param request
+     * @param isRequesting
+     * @param localIP
+     * @param webContextProperties
+     * @param remoteObjectIP
+     */
+    protected void testGetActiveParticipantDestination(LogEventRequestType request, Boolean isRequesting, String localIP,
+        Properties webContextProperties, String remoteObjectIP) {
```

Comment:
In the future, favor adding an extra line break if the method declaration is wrapped -- but DON'T worry about going back and making this change for the current PR.